src/SecretSanta/Models.elm

```diff
@@ -0,0 +1,22 @@
+module SecretSanta.Models exposing (..)
+
+
+type alias SecretSanta =
+    { date : String
+    , pairs : List Pair
+    }
+
+
+type alias Pair =
+    { id : String
+    , santa : Person
+    , giftee : Person
+    }
+
+
+type alias Person =
+    { user_id : String
```

Comment:
@mbithenzomo Since it's a minor fix, we could have it changed on the next secret santa story? I'm going to go right ahead and merge this.